WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -15,12 +15,14 @@ msgstr ""
 
 #. translators: The application name in the Apple App Store. Please keep the brand names ('Jetpack' and WordPress') verbatim. Limit to 30 characters including spaces and punctuation!
 msgctxt "app_store_name"
-msgid "Jetpack – Website Builder"
+msgid "Jetpack – Website Builder
+"
```

Comment:
🤔 I'm not sure about these changes...

This is the first time I generated the `.po` with release toolkit v9.0.0, but nothing in [its changelog](https://github.com/wordpress-mobile/release-toolkit/releases/tag/9.0.0) stands out as a possible cause for this.

Reply:
I'm glad with have `gplint` in place. I have a feeling this wasn't valid but didn't remember why. [The linter confirmed it](https://buildkite.com/automattic/wordpress-ios/builds/16129#0189e28a-72ae-439c-9003-545f8af6904d/175-195).

Reply:
> but nothing in [its changelog](https://github.com/wordpress-mobile/release-toolkit/releases/tag/9.0.0) stands out as a possible cause for this.

Update: I bet it's the PR that renamed the metadata parsing files, https://github.com/wordpress-mobile/release-toolkit/pull/497. The name was hidden in the "Support Ruby 3.0" umbrella.